plugins/couchbase.yaml

```diff
@@ -167,8 +167,6 @@ pipeline:
       layout: '%Y-%m-%dT%H:%M:%S.%L%j'
     severity:
       parse_from: couchbase_severity
-      mapping:
-        warning: warn
```

Comment:
The ones that will definitely cause the most problems will be "notice" and "critical". One shotgun approach we could take to solve the implicit problem is explicitly call out all of the deprecated types and map them to a supported type in each plugin. It might look a little silly in plugins where the source never returns that severity, but we should be able to make judgement calls on when not to do it.